src/utils/data/data_01.js

```diff
@@ -0,0 +1 @@
+// this file is a sample file for data files
```

Comment:
I don't understand. What does this mean? What's supposed to be inside of these files? How do we use data files in the code?

It's fine to have this file here, but then this file (or maybe the README file for this repo?) should provide me with enough information that I can use it for something.